src/lightly_train/_task_models/dinov3_ltdetr_object_detection/task_model.py

```diff
@@ -775,3 +780,117 @@ def export_onnx(
                 )
 
         logger.info(f"Successfully exported ONNX model to '{out_path}'")
+
+    def export_tensorrt(
+        self,
+        onnx_path: PathLike,
+        engine_path: PathLike,
```

Comment:
```suggestion
        out: PathLike,
```

Let's call the output path  `out` as in the other functions like `train`, `export`, `predict`, etc. I would also change it for `export_onnx` (sorry didn't notice that in that PR)